File: main/java/com/rperazzo/weatherapp/Model/WeatherService.java
package com.rperazzo.weatherapp.Model;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface WeatherService {

      @GET("find")
    Call<FindResult<City>> find(
            @Query("q") String cityName,
            @Query("units") String units,
            @Query("appid") String apiKey
    );
}

File: weather-app/app/src/main/java/com/rperazzo/weatherapp/model/service/WeatherService.kt
package com.rperazzo.weatherapp.model.service

import com.rperazzo.weatherapp.model.entity.FindResult
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

public interface WeatherService {
    @GET("find")
    fun find(
            @Query("q") cityName: String,
            @Query("units") units: String,
            @Query("appid") apiKey: String
    ): Call<FindResult>;
}
